NewForce-Cohort-9/tabloidmvc-josh-jacobs-jingleheimer-schmidt
# Danielbranch1

# Description

Added a view for user profile. 
As an admin, I would like to see a list of all the User Profiles so that I can keep track of who is using the system.





## Type of change





# Testing Instructions

1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout  danielbranch1<danielbranch1>
5. If you don't see the most recent version of my files, git pull origin danielbranch1 <danielbranch1>
6. If you haven't created the Tabloid MVC database, please do so.
7. Use this to fill out tables and columns: SQL data
8. Use this to seed the database: SQL Seed Data
9. Click the execute with debugger solid green button to launch the app
10. In browser, please go to Login on the nav bar and log in with this email: admin@example.com
11. Navigate to User Profiles 
12. Check to see that the FullName, DisplayName and UserType Id are displayed to the broser in that order from Left to right. 
13. Look to see if you have the option to edit and delete a user profile. There should also be a link that gives you the option  to view the details *I have not done the details yet so there shouldn't be anything there if you click on it*
14. The Navbar should also be displayed at the top of the web page. 
15. There should be only one user profile displayed: Admina Strator and her display name should be admin and she should have a UserTypeId of 1. 


# Checklist:
